src/react/features/rooms/effects/useGetHomeEffect.js

```diff
@@ -7,12 +7,22 @@ import { setConnections } from '../../connectors/actions'
 
 import { invokeChannel } from '../../../client/client'
 import { channels } from '../../../../shared/constants'
-
+import { invokeAlertDialog } from '../../../client/alertDialog'
+/*
+ * Requests home & focus info via GET_HOME and sets the focus for room & diory
+ * - saves the response to the store
+ * - invoke an alert for each error
+ */
 export const useGetHomeEffect = () => {
   const dispatch = useDispatch()
 
   useEffect(() => {
-    invokeChannel(channels.GET_HOME).then(({ rooms, connections, focus }) => {
+    invokeChannel(channels.GET_HOME).then(({ rooms, connections, focus, errors }) => {
+      errors.forEach((error) => {
+        invokeAlertDialog(
+          `Couldn't connect to room ${error.connectionURI}, path didn't exist. Please remove and re-add the room to reconnect.`
+        )
+      })
```

Comment:
We would add error feature with actions, reducer and components. 
Then here dispatch(addErrors(errors)))